dsb-gui/src/main/java/com/spectralogic/dsbrowser/gui/Main.java

```diff
@@ -71,17 +97,17 @@
             exitButton.setText("Exit");
             cancelButton.setText("Cancel");
 
-            if (jobWorkers.getTasks().size() == 1) {
-                CLOSECONFIRMATIONALERT.setHeaderText(jobWorkers.getTasks().size() + " job is still running.");
+            if (notCachedRunningTasks.size() == 1) {
+                CLOSECONFIRMATIONALERT.setHeaderText(notCachedRunningTasks.size() + " job is still running.");
             } else {
-                CLOSECONFIRMATIONALERT.setHeaderText(jobWorkers.getTasks().size() + " jobs are still running.");
+                CLOSECONFIRMATIONALERT.setHeaderText(notCachedRunningTasks.size() + " jobs are still running.");
             }
 
             final Optional<ButtonType> closeResponse = CLOSECONFIRMATIONALERT.showAndWait();
 
             if (closeResponse.get().equals(ButtonType.OK)) {
-                closeApplication();
-                event.consume();
+                closeApplication(event);
+                /*event.consume();*/
```

Comment:
Unless this is planned on being added back this commented out code should be removed.